### PR TITLE
rework Zinnia errors reported to Sentry

### PR DESCRIPTION
**Pull request by bajtos**
- Add the module name & the exit code to the message
- Remove Zinnia & module main file path from the message

The goal is to allow Sentry to group all errors with the same cause in a single issue.

Add the last 50 lines from the Execa error message (including stdout & stderr) into the `extra` fields sent to Sentry.

This is a follow-up for https://github.com/filecoin-station/core/pull/377

### CODE REVIEW

**Review by juliangruber — APPROVED** [review]
great thinking!

**Comment by bajtos** [comment]
Example console log after running `killall zinniad`:

```
[19/03/2024, 17:39:11] INFO  Spark exited via signal SIGTERM
[19/03/2024, 17:39:11] ERROR Spark crashed via signal SIGTERM
[19/03/2024, 17:39:11] INFO  Voyager exited via signal SIGTERM
Zinnia closed all stdio with codes SIGTERM, SIGTERM
```

Example Sentry issue - search for "Additional data" section and the key "details": https://protocol-labs-ip.sentry.io/issues/5083788454/?project=4504792315199488&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=5

Screenshot 2024-03-19 at 17 41 01

**Comment by bajtos** [comment]
I made quite a few adjustments. Could you please review my changes again, @juliangruber?

File: lib/zinnia.js [hunk 1]
```diff
@@ -228,7 +245,7 @@ const waitForStdioClose = async ({ childProcesses, controller }) => {
   const results = await Promise.all(
     childProcesses.map(childProcess => once(childProcess, 'close'))
   )
-  const codes = results.map(([code]) => code ?? '<no code>')
+  const codes = results.map(([code, signal]) => code || signal || '<no code>')
```
Comment (juliangruber) [inline lib/zinnia.js:248]:
```suggestion
  const codes = results.map(([code, signal]) => code || `<${signal}>` || '<no code>')
```

I think through this the log line will be clearer, as a signal is a special "code", like "no code"

**Review by juliangruber — APPROVED** [review]

File: lib/zinnia.js (hunk 1 above)
Comment (bajtos) [reply lib/zinnia.js:248]: Makes sense 👍🏻

**Comment by bajtos** [comment]
New logs:

```
[20/03/2024, 09:53:06] INFO  Spark exited via signal SIGTERM
[20/03/2024, 09:53:06] ERROR Spark crashed via signal SIGTERM
Reporting the poblem to Sentry for inspection by the Station team.
[20/03/2024, 09:53:06] INFO  Voyager exited via signal SIGTERM
Zinnia closed all stdio with codes <SIGTERM>, <SIGTERM>
```

**Pull request merged**